src/entry.cpp

```diff
@@ -44,6 +49,40 @@ extern "C" __declspec(dllexport) AddonDefinition* GetAddonDef()
 	return &G::AddonDef;
 }
 
+Keybind ParseArcDPSKeybind()
+{
+	const auto arcIniPath = std::filesystem::path(G::APIDefs->Paths.GetAddonDirectory("arcdps")) / "arcdps.ini";
+	G::ArcOptions.global_mod1 = GetPrivateProfileIntA("keys", "global_mod1", VK_SHIFT, arcIniPath.string().c_str());
+	G::ArcOptions.global_mod2 = GetPrivateProfileIntA("keys", "global_mod2", VK_MENU, arcIniPath.string().c_str());
+	G::ArcOptions.global_options = GetPrivateProfileIntA("keys", "global_options", 'T', arcIniPath.string().c_str());
+	return Keybind{
+		.Key = static_cast<unsigned short>(G::ArcOptions.global_options),
```

Comment:
Might have messed something up in here, since `T` gets displayed as `ISO_Level3_Shift` - but this is display only bug since anyway it gets handled by Arc and not Nexus.

Reply:
This is because Nexus handles scancodes while ArcDPS uses Virtual Keys, but it's easy to fix.